module/x/gravity/types/msgs.go

```diff
@@ -92,7 +92,7 @@ func (msg *MsgSubmitEthereumEvent) ValidateBasic() (err error) {
 
 // GetSignBytes encodes the message for signing
 func (msg *MsgSubmitEthereumEvent) GetSignBytes() []byte {
-	panic(fmt.Errorf("deprecated"))
+	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(msg))
```

Comment:
This would be a good opportunity to standardize the receivers on using a value rather than pointer.